internal/util/mqclient/rmq_client.go

```diff
@@ -45,6 +45,11 @@ func (rc *rmqClient) CreateProducer(options ProducerOptions) (Producer, error) {
 	return &rp, nil
 }
 
+//TODO fishpenguin: implementation
+func (rc *rmqClient) CreateReader(options ReaderOptions) (Reader, error) {
+	panic("this method has not been implented")
```

Comment:
Please implement this function before actually using MsgStream Reader functions

Reply:
lgtm